### DISCUSSION TITLE
scRNA-seq as pseudobulk for DEseq2: does AggregateExpression() normalize counts?

### DISCUSSION THREAD

**Discussion by DaianeH** [satijalab/seurat#8201] [General]
Hi,

I'm using AggregateExpression() function to convert my scRNA-seq data into pseudobulk for differential expression with Deseq2. I'm wondering whether AggregateExpression() simply sums the counts for each gene in each cell, or if it also normalizes by the different numbers of cells that each sample has.

This is the Deseq2 plot for gene LRP6 showing it is differentially expressed in group MUT as compared to WT. Within the group MUT, however, I am wondering whether the sample highlighted in red has higher expression of this gene simply because it may have more cells (let's say for example 10,000), and the sample highlighted in blue has lower expression of this gene because it may have way less cells (let's say for example 500).

![Slide1](https://github.com/satijalab/seurat/assets/2428375/c3a48e21-20e5-455f-9cc0-b574cf857614)

Could that be the case? And if so, how to avoid it?

**Comment by RaghadShu** [comment]
Any answers to this question?
